src/citycoin-client.ts

```diff
@@ -0,0 +1,204 @@
+import { Tx, Chain, Account, types } from 'https://deno.land/x/clarinet@v0.6.0/index.ts';
+
+//used only for better intellisense 
+export interface Result {
+  session_id: number,
+  result: string
+  events: []
+}
+
+export class CityCoinClient {
+  contractName: string = "citycoin"
+  chain: Chain;
+  deployer: Account;
+
+  constructor(chain: Chain, deployer: Account) {
+    this.chain = chain;
+    this.deployer = deployer;
+  }
+
+  private callReadOnlyFn(method: string, args: Array<any> = [], sender: Account = this.deployer): Result {
+    const result = this.chain.callReadOnlyFn(
+      this.contractName,
+      method,
+      args,
+      sender?.address
+    );
+
+    return result;
+  }
+
+  /**
+   * Mints token to make testing easier.
+   * 
+   * @param amount 
+   * @param recipient 
+   */
+  ftMint(amount: number, recipient: Account) {
+    const block = this.chain.mineBlock([
```

Comment:
should be without `mineBlock` wrap

Reply:
You're right. I corrected it with  6a6f727